### ISSUE TITLE
allDocs service and allDocs controller

### ISSUE THREAD

**Issue by mohammadhafiz** [pouchdb/pouchdb#4548]
hai can u answer my question. 
this is my service.

application.service('Arrears', [
function() {
var db = new PouchDB('localhost:5984');
return {
put: function (user) {
var data = {
"_id": new Date().toString(),
"account_no": user.account_no,
"address": user.address,
"arrears": user.arrears,
"cases_id": user.cases_id,
"contact_no": user.contact_no,
"council": user.council,
"detail": user.detail,
"giant_id": user.giant_id,
"land_cass": user.land_cass,
"last_paid_ammount": user.last_paid_ammount,
"lastpayDate": user.lastpayDate,
"location": user.location,
"owner_id": user.owner_id,
"owner_name": user.owner_name,
"paid_until": user.paid_until,
"property_id": user.property_id,
"receipt_no": user.receipt_no,
"status": user.status,
"title_no": user.title_no,
};
return db.put(data);
},
get: function (user) {
db.get(data)
},
allDocs: function (user) {
var data = {
"_id": user.rows[i].doc._id,
"account_no": user.rows[i].doc.account_no,
"address": user.rows[i].doc.address,
"arrears": user.rows[i].doc.arrears,
"cases_id": user.rows[i].doc.cases_id,
"contact_no": user.rows[i].doc.contact_no,
"council": user.rows[i].doc.council,
"detail": user.rows[i].doc.detail,
"giant_id": user.rows[i].doc.giant_id,
"land_cass": user.rows[i].doc.land_cass,
"last_paid_ammount": user.rows[i].doc.last_paid_ammount,
"lastpayDate": user.rows[i].doc.lastpayDate,
"location": user.rows[i].doc.location,
"owner_id": user.rows[i].doc.owner_id,
"owner_name": user.rows[i].doc.owner_name,
"paid_until": user.rows[i].doc.paid_until,
"property_id": user.rows[i].doc.property_id,
"receipt_no": user.rows[i].doc.receipt_no,
"status": user.rows[i].doc.status,
"title_no": user.rows[i].doc.title_no
}
return db.allDocs(data)
}
}
}
]);

can u give me code for me to call allDocs service into controller.

application.controller('CreateManagementController', ['$mdSidenav', '$location', '$scope', 'Arrears',
function($mdSidenav, $location, $scope, Arrears)
{
$scope.limit = 50;
$scope.page = 0;
$scope.theme = 'teal';
$scope.put = function(user)
{
Arrears.put(user)
.then(function(response) 
{
console.info(response);
}).catch(function(error) 
{
console.error(error);
})
};
$scope.menu = function()
{
$mdSidenav('menu').open();
};
$scope.reset = function()
{
$scope.query.$ = '';
};
}]);

**Comment by daleharvey** [comment]
Sorry it seems like your question is more related to angular or whatever framework you are using, allDocs in pouchdb is called with `db.allDocs().then(...`

**Comment by nickcolley** [comment]
Yo @mohammadhafiz if this is related to PouchDB specifically can you rephrase the issue to make it a bit clearer?

If not like Dale said, probably better off trying http://stackoverflow.com/

I'm going to close this out but can reopen if you think you've got a PouchDB specific issue :+1:
